integration-tests/ci-visibility-intake.js

```diff
@@ -169,6 +169,22 @@ class FakeCiVisIntake extends FakeAgent {
     infoResponse = DEFAULT_INFO_RESPONSE
   }
 
+  gatherPayloads (payloadMatch, gatheringTime = 15000) {
+    const payloads = []
+    return new Promise((resolve, reject) => {
+      setTimeout(() => {
+        this.removeListener('message', messageHandler)
```

Comment:
in the same method, can we use the same syntax? I mean: `removeListener` with `addListener` or `on` with `off`

Reply:
yeah that's a good idea